<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Account_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="Register_Label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Account_LineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_LineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Account_LineEdit</tabstop>
  <tabstop>Password_LineEdit</tabstop>
  <tabstop>LoginBtn</tabstop>
  <tabstop>CancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
